Source/Core/PointGeometry.js

```diff
@@ -50,6 +50,9 @@ define([
         if (!defined(options.positionsTypedArray)) {
             throw new DeveloperError('options.positionsTypedArray is required.');
         }
+        if (!defined(options.colorsTypedArray)) {
```

Comment:
This is what's in master, I had removed it when I was still using the `PointGeometry` to render `Points3DTileContent`, but since that isn't the case anymore I reverted all the changes.